### DISCUSSION TITLE
API downloaded data doesn't add up?

### DISCUSSION THREAD

**Discussion by bodong408** [electricitymaps/electricitymaps-contrib#5418] [Q&A] [answered]
Hi everyone, I am trying to validate EM's API downloaded data. My question is: what is the correct formula to roll up power_consumption_breakdown.* into power_consumption_total? I see some mismatches between sum(powerConsumptionBreakdown.*) vs. powerConsumptionTotal. 

For example: zone = 'IT' and datetime = '2021-01-02T22:00:00'.
Another example: zone = 'CA-PE' and datetime = '2017-05-15T18:00:00'

Is this formula correct?
calc_Consumption_total =?=
      powerConsumptionBreakdown.nuclear
    + powerConsumptionBreakdown.geothermal
    + powerConsumptionBreakdown.biomass
    + powerConsumptionBreakdown.coal
    + powerConsumptionBreakdown.wind
    + powerConsumptionBreakdown.solar
    + powerConsumptionBreakdown.hydro
    + powerConsumptionBreakdown.gas
    + powerConsumptionBreakdown.oil
    + powerConsumptionBreakdown.unknown
    + powerConsumptionBreakdown.hydro_discharge
    + powerConsumptionBreakdown.battery_discharge

**Comment by VIKTORVAV99** [comment, marked as answer]
This topic has been discussed more in depth on our slack server and the issue have been addressed.
